medical/models/res_partner.py

```diff
@@ -48,26 +32,31 @@ class ResPartner(models.Model):
     alias = fields.Char(
         string='Alias',
         size=256,
-        help='Common name that the Party is reffered',
+        help='Common name the party is referred to as',
     )
     activation_date = fields.Date(
-        string='Activation date',
-        help='Date of activation of the party',
+        string='Activation Date',
+        help='Date the party was activated',
     )
     last_name = fields.Char(
```

Comment:
Removed. Added "Separation of patient/res_partner name into first and last names." to roadmap in readme.